code/sonalyze/db/sysinfo_v0json_parser.go

```diff
@@ -0,0 +1,49 @@
+// Parser for v0 "new format" JSON files holding Sonar `sysinfo` data.
+
+package db
+
+import (
+	"io"
+
+	"github.com/NordicHPC/sonar/util/formats/newfmt"
+	"go-utils/config"
+)
+
+// If an error is encountered we will return the records successfully parsed before the error along
+// with an error, but there is no ability to skip erroneous records and continue going after an
+// error has been encountered.
+//
+// FIXME: This is a really lossy translation.  First, the old SysinfoEnvelope can't hold all the
```

Comment:
Remove "FIXME" and replace with "NOTE", this will be updated in a subsequent queue, already planned.